src/redux/admin/admin.reducer.js

```diff
@@ -0,0 +1,46 @@
+import {
+  SET_AUTH,
+  SET_ADMIN_ERROR,
+  SET_ADMIN_LOADING,
+  LOGOUT_ADMIN
+} from './admin.types';
+
+const initialState = {
+  isAuth: null,
+  adminError: null,
```

Comment:
Useless prefix `admin` here. We don't need this kind of prefixes for the keys that always will be isolated inside its reducer. Some type of prefix we need only for the action types because it is dispatched globally like `SET_NEWS_ERROR` and `SET_AUTH_ERROR` are two different actions and are handled by two different reducers that's why we need to have two different action types